chapters/id/_toctree.yml

```diff
@@ -0,0 +1,201 @@
+- title: 0. Persiapan
+  sections:
+  - local: chapter0/1
+    title: Pendahuluan
+
+- title: 1. Model-model Transformer
+  sections:
+  - local: chapter1/1
+    title: Pendahuluan
+  - local: chapter1/2
+    title: Pemrosesan Bahasa Natural
+#   - local: chapter1/3
```

Comment:
I suggest removing these commented out sections to avoid some merge conflicts if others in the community happen to work on the translation in parallel